.github/CODEOWNERS

```diff
@@ -2122,4 +2122,5 @@ spec/requests/va_profile/contacts_request_spec.rb @department-of-veterans-affair
 /.github/workflows/build-and-publish.yaml @department-of-veterans-affairs/backend-review-group
 modules/accredited_representative_portal @department-of-veterans-affairs/benefits-accredited-rep-facing-engineers @department-of-veterans-affairs/backend-review-group
 README.md @department-of-veterans-affairs/backend-review-group
+config/form_profile_mappings/FORM-MOCK-AE-DESIGN-PATTERNS.yml @department-of-veterans-affairs/tmf-auth-exp-design-patterns
```

Comment:
I think this is fine, but we probably will need to have our github team added to the repo like we did for the FE pr of a similar fashion.